get-started/part4.md

```diff
@@ -67,35 +67,41 @@ machine a swarm manager. From then on, Docker will run the commands you execute
 on the swarm you're managing, rather than just on the current machine.
 
 {% capture local-instructions %}
-You now have two VMs created, named `myvm1` and `myvm2` (as `docker-machine ls`
-shows). The first one will act as the manager, which executes `docker` commands
-and authenticates workers to join the swarm, and the second will be a worker.
+You now have two VMs created, named `myvm1` and `myvm2`:
+
+```shell
+docker-machine ls
+```
+
+The first one will act as the manager, which executes management commands and
+authenticates workers to join the swarm, and the second will be a worker.
 
 You can send commands to your VMs using `docker-machine ssh`. Instruct `myvm1`
 to become a swarm manager with `docker swarm init` and you'll see output like
 this:
 
 ```shell
-$ docker-machine ssh myvm1 "docker swarm init"
+$ docker-machine ssh myvm1 "docker swarm init --advertise-addr <myvm1 ip>"
```

Comment:
The tutorial desperately needs updated with the correct procedure. Because of the default `ls` output. The text at https://docs.docker.com/get-started/part4/ still maintains, as a default procedure, we should specify port 2376 when initializing a swarm manager... Despite the short quip about default management ports, the tutorial instructions can be confusing and frustrating to new users. I suggest the example command be this:

`docker-machine ssh myvm1 "docker swarm init --advertise-addr <myvm1 ip:2377>"`

Thus the note following about management ports will be intuitive.